views/pulsar/components/tab.html.twig

```diff
@@ -7,26 +7,39 @@
 
 {% set _hasSettings = false %}
 {% set _hasSidebar = false %}
+{% set _actions_left = '' %}
+{% set _actions_right = '' %}
 
-{% set _settings = block('tab_settings') %}
-{% if _settings is not empty %}
-    {% set _hasSettings = true %}
+{% if block('tab_settings') is defined %}
+    {% set _settings = block('tab_settings') %}
+    {% if _settings is not empty %}
+        {% set _hasSettings = true %}
+    {% endif %}
+{% endif %}
+
+{% if block('tab_sidebar') is defined %}
+    {% set _sidebar = block('tab_sidebar') %}
+    {% if _sidebar is not empty %}
+        {% set _hasSidebar = true %}
+    {% endif %}
 {% endif %}
 
-{% set _sidebar = block('tab_sidebar') %}
-{% if _sidebar is not empty %}
-    {% set _hasSidebar = true %}
+{% if block('tab_js') is defined %}
```

Comment:
Might be worth just removing support for `tab_js` block since we're making a breaking change release anyway?